package.json

```diff
@@ -65,13 +65,15 @@
     "js-yaml": "3.13.1",
     "lodash": "4.17.15",
     "ltx": "2.9.2",
+    "mini-css-extract-plugin": "^0.9.0",
     "npm-run-all": "4.1.5",
     "postcss-inline-svg": "4.1.0",
     "postcss-loader": "3.0.0",
     "prettier": "2.0.5",
     "remark-preset-lint-recommended": "4.0.0",
     "remark-preset-prettier": "0.4.0",
     "require-dir": "1.2.0",
+    "resolve-url-loader": "^3.1.1",
```

Comment:
I don't see this referenced in our code, is it still required or is it an artifact of an earlier exploration?

Reply:
> artifact of an earlier exploration?

You got it. Removing